extensions/data-transfer/portability-data-transfer-rememberthemilk/src/main/java/org/dataportabilityproject/transfer/rememberthemilk/tasks/RememberTheMilkTasksImporter.java

```diff
@@ -79,10 +83,12 @@ public ImportResult importItem(UUID jobId, AuthData authData, TaskContainerResou
       for (TaskModel task : data.getTasks()) {
         String newList = tempTasksData.lookupNewTaskListId(task.getTaskListId());
         TaskSeries addedTask = service.createTask(task.getText(), timeline, newList);
-        //todo: add notes
+        // todo: add notes
       }
-    } catch (IOException e) {
-      return new ImportResult(ImportResult.ResultType.ERROR);
+    } catch (Exception e) {
+      logger.warn("Error importing item: " + e.getMessage());
+      logger.warn(Throwables.getStackTraceAsString(e));
```

Comment:
Remove this second one since the message is above?

Reply:
Done.